content/docs/conceptual-guides/branching.md

```diff
@@ -6,16 +6,49 @@ redirectFrom:
 
 <a id="branches-coming-soon/"></a>
 
-_Neon Branching capabilities are not yet publicly available yet. If you would like to try this feature, reach out to [iwantbranching@neon.tech](iwantbranching@neon.tech) describing your use case and requesting that Neon  enable branching capabilities for your account.
+A branch is a copy of your Neon Project data created from the current state or from a past state that is still available.
 
-A branch is a copy of the project data created from the current state or any past state that is still available. A branch can be independently modified from its originating project data.
+When you create a branch, all of the data in the parent Project is available in the branched Project, but changes to the branch afterward are independent of the parent Project and vice versa.
```

Comment:
current behaviour: creating a branch creates a new project
future behaviour: creating a branch will create an additional branch of the project

'available in the branched Project' this line implies the current behavior, but not the future behavior.
can we reword to avoid this implication?